File: deal/src/main/java/ru/artemev/deal/model/enums/MaritalStatus.java
package ru.artemev.deal.model.enums;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;

@Schema(description = "Семейное положение")
@AllArgsConstructor
public enum MaritalStatus {
  MARRIED("Замужем/женат"),
  DIVORCED("В разводе"),
  SINGLE("Холост"),
  WIDOW_WIDOWER("Вдова");

  private final String description;
}

File: gateway/src/main/kotlin/ru/artemev/gateway/model/enums/MaritalStatus.kt
package ru.artemev.gateway.model.enums

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Семейное положение")
enum class MaritalStatus(val description: String) {
    MARRIED("Замужем/женат"),
    DIVORCED("В разводе"),
    SINGLE("Холост"),
    WIDOW_WIDOWER("Вдова")
}